src/containers/Root.js

```diff
@@ -20,8 +20,12 @@ export default class Root extends React.Component {
   get devTools () {
     if (__DEBUG__) {
       if (__DEBUG_NEW_WINDOW__) {
-        require('../redux/utils/createDevToolsWindow')(this.props.store)
-      } else {
+        if (!window.devToolsExtension) {
+          require('../redux/utils/createDevToolsWindow')(this.props.store)
+        } else {
+          window.devToolsExtension.open()
+        }
+      } else if (!window.devToolsExtension) {
```

Comment:
What's the purpose of this check on line 28 if the extension tools are only being used if `dev:nw` is active? To me it would be more intuitive to allow the extension to _always_ override the local tools, whether or not it's as a sidebar or new window.